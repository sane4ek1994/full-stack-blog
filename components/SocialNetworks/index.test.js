import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { SocialNetworks } from './index'

it('expect to render SocialNetworks component', () => {
  const SocialNetworksComponent = shallow(<SocialNetworks />)

  expect(toJson(SocialNetworksComponent)).toMatchSnapshot()
})
