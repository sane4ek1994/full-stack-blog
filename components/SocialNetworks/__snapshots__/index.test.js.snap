// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SocialNetworks component 1`] = `
<ScreenEgg>
  <ul
    className="list"
  >
    <li
      className="item"
      key="1"
    >
      <a
        className="link"
        href="https://github.com/sane4ek1994"
        rel="noreferrer"
        target="_blank"
      >
        <AiFillGithub
          color="black"
          size={50}
        />
      </a>
    </li>
    <li
      className="item"
      key="2"
    >
      <a
        className="link"
        href="https://www.youtube.com/"
        rel="noreferrer"
        target="_blank"
      >
        <AiFillYoutube
          color="black"
          size={50}
        />
      </a>
    </li>
    <li
      className="item"
      key="3"
    >
      <a
        className="link"
        href="https://www.twitter.com/"
        rel="noreferrer"
        target="_blank"
      >
        <AiFillTwitterCircle
          color="black"
          size={50}
        />
      </a>
    </li>
    <li
      className="item"
      key="4"
    >
      <a
        className="link"
        href="https://www.instagram.com/"
        rel="noreferrer"
        target="_blank"
      >
        <AiFillInstagram
          color="black"
          size={50}
        />
      </a>
    </li>
    <li
      className="item"
      key="5"
    >
      <a
        className="link"
        href="https://www.linkedin.com/"
        rel="noreferrer"
        target="_blank"
      >
        <AiFillLinkedin
          color="black"
          size={50}
        />
      </a>
    </li>
  </ul>
</ScreenEgg>
`;
